### ISSUE TITLE
A11y: In Time Picker Calendar the Tab is used where the arrows Up and Down should be used instead

### ISSUE THREAD

**Issue by polibb** [grafana/grafana#56235]
**What happened**:
In the **calendar picker** inside the Time Picker on a dashboard, moving with arrow Up/Down/Left/Right is not doing anything. We are using tab to preselect the day/month/year option we are aiming at. 

**What you expected to happen**:
Inside the list of what we want to select in the calendar as day/month/year, we should be able use the arrows Up/Down/Left/Right to get around for a faster selection.

**How to reproduce it (as minimally and precisely as possible)**:
Open Time Picker in a dashboard. Open a calendar to select a date option. Get to the selection of a day/month/year with the Tab. Inside the selection of what you want, use the arrows Up/Down/Left/Right to get to the specific selection you want. Nothing happens, moving is only possible with a Tab.

https://user-images.githubusercontent.com/13227501/193759581-2ba73923-9f08-4993-952a-015e037afcbb.mov

**Anything else we need to know?**:

**Relevant WCAG Criteria**:
[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 9.3.0
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:
